SPECS/python-cherrypy/python-cherrypy.spec

```
@@ -6,8 +6,8 @@
 
 Summary:        A pythonic, object-oriented HTTP framework
 Name:           python-%{pkgname}
-Version:        18.6.1
-Release:        3%{?dist}
+Version:        18.9.0
+Release:        1%{?dist}
 License:        BSD
 Url:            https://cherrypy.org/
```

I don't think this url is correct, IMHO it should be:
```suggestion
Url:            https://cherrypy.dev/
```

---

Thank you! will fix this